src/traces/parcoords/parcoords.js

```diff
@@ -557,7 +557,7 @@ module.exports = function(root, svg, styledData, layout, callbacks) {
                     .outerTickSize(2)
                     .ticks(wantedTickCount, d.tickFormat) // works for continuous scales only...
                     .tickValues(d.ordinal ? // and this works for ordinal scales
-                        sdom.map(function(d, i) {return texts && texts[i] || d;}) :
+                        sdom.map(function(d, i) {return texts && texts[i] !== undefined ? texts[i] : d;}) :
```

Comment:
@alexcjohnson to summarise if I understand your latest comments and Étienne's comment on the top, this PR needs these: 

1. in `parcoords`, if `ticktext[i]` is either `undefined` or `null`, then we consider it unsupplied, ie. revert to just using `tickvals[i]` (the current PR currently checks for `undefined` only but I can extend)
2. `tickvals[i]` should be subject to `tickformat` as on my above screenshot with the percentages in column 3 and 4.
3. these things should be locked down in test so that, when we get around to axis unification, we can detect regressions
4. I'd create an issue that refers to comments in this PR that would be the basis for considering a change in cartesian such that it also does what's described in point 1 of this list